rakelib/sync.rake

```diff
@@ -0,0 +1,31 @@
+namespace :sync do
+  REPOSITORIES = [:StencilSwiftKit, :SwiftGenKit, :SwiftGen, :templates]
+
+  desc 'Synchronize all files across repositories'
+  task :all => [:rakelib, :gitignore, :license]
+
+  task :gems do |task|
+    REPOSITORIES.each do |repository|
+      FileUtils.cp('common/Gemfile', "#{repository}/")
+      FileUtils.cp('common/Gemfile.lock', "#{repository}/")
```

Comment:
🤔 Shouldn't we run a `bundle install` automatically after overriding the `Gemfile.lock`?